scripts/set_kokkos_env.sh

```diff
@@ -0,0 +1,9 @@
+# set nvcc as the underlying C++ compiler for Open MPI
+export OMPI_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
+
+# set nvcc as the underlying C++ compiler for MPICH
+export MPICH_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
```

Comment:
I think the preferred way is to use  `nvcc_wrapper` in `bin/nvcc_wrapper` instead of `config/nvcc_wrapper` (kokkos/kokkos#562). I don't think it's too important now, though.